NachoSpigot-Server/src/main/java/net/minecraft/server/DedicatedServer.java

```diff
@@ -467,6 +459,12 @@ public boolean ai() {
     }
 
     public DedicatedPlayerList aP() {
+        // Nacho start - OBFHELPER
+        return getDedicatedPlayerList();
+    }
+
+    public DedicatedPlayerList getDedicatedPlayerList() {
```

Comment:
getDedicatedPlayerList should call aP as aP is a hot function, i know it gets inlined but no need make it wait